src/CoreBundle/Resources/public/themes/standard/css/global.css

```diff
@@ -529,19 +446,15 @@ input, textarea, select,
     padding: 5px;
 }
 
-.navbar-brand {
-    padding: 5px 0 17px 15px;
-    margin-right: 30px;
+#header-nav {
+  padding: 0;
 }
 
-.navbar-default .navbar-nav, .navbar-inverse .navbar-nav {
-    background: transparent!important;
-}
-
-.navbar-inverse {
-    background: #404040!important;
+#ToggleRecording {
```

Comment:
Can we somehow avoid using specific values for single elements and bind those rules to classes?